src/shell/notifications.test.js

```diff
@@ -29,42 +29,6 @@ describe('notifications', () => {
     expect(within(notification).getByText('Carrot Soup')).toBeInTheDocument()
   })
 
-  test('notification is shown when cow pen is purchased', async () => {
-    const loadedState = saveDataStubFactory({
-      money: 1500,
-    })
-
-    await farmhandStub({
-      localforage: {
-        getItem: () => Promise.resolve(loadedState),
-        setItem: (_key, data) => Promise.resolve(data),
-      },
-    })
-
-    await nextView()
-    const upgradesTab = screen.getByText('Upgrades')
-    userEvent.click(upgradesTab)
-
-    const cowPenContainer = screen
-      .getByText('Buy cow pen')
-      .closest('.TierPurchase')
-
-    // Open the list of cow pen options
-    userEvent.click(within(cowPenContainer).getAllByRole('button')[1])
-
-    userEvent.click(screen.getByRole('option', { name: '$1,500: 10 cow pen' }))
-
-    // Make the purchase
-    userEvent.click(within(cowPenContainer).getAllByRole('button')[0])
-
-    const notification = await screen.findByRole('alert')
-    expect(
-      within(notification).getByText(
-        'Purchased a cow pen with capacity for 10 cows! You can visit your cow pen by going to the "Cows" page.'
-      )
-    ).toBeInTheDocument()
-  })
```

Comment:
This is effectively covered by this new test, which is much simpler and more lightweight: https://github.com/jeremyckahn/farmhand/blob/73056d81491eedb255dccdf698ad54c01729df08/src/game-logic/reducers/purchaseCowPen.test.js#L27-L35